source/Calamari/Deployment/Conventions/LegacyIisWebSiteConvention.cs

```diff
@@ -49,6 +49,32 @@ public void Install(RunningDeployment deployment)
             Log.Info("The IIS website named '{0}' has had its path updated to: '{1}'", iisSiteName, webRoot);
         }
 
+        static bool LegacySupport(RunningDeployment deployment)
+        {
+            var legacySupport = deployment.Variables.GetFlag(SpecialVariables.UseLegacyIisSupport);
+
+            // IIS7 was shipped in Windows Server 2008 and so this should not be getting hit by ANY customers (i.e. 2003 is deprecated)
+            if (!legacySupport)
+                return false;
+
+            // We were previously never warning of its imminent removal so
+            // provide a final legacy support capability with another big warning
+            // This will provide us some cover for later removal.
+            var forceLegacySupport = deployment.Variables.GetFlag(SpecialVariables.UseLegacyIisSupportForce);
+            if (!forceLegacySupport)
+                throw new CommandException($"Support for IIS6 is no longer supported.\r\n"
+                                           + $"Remove the {SpecialVariables.UseLegacyIisSupportForce} variable and ensure you are targeting IIS7+.\r\n"
+                                           + $"To provide a temporary work around set the `{SpecialVariables.UseLegacyIisSupportForce}` to `true`.\r\n"
+                                           + $"This capability will be very shortly removed without further warning.");
```

Comment:
This doesn't make sense to me. We are telling them to  remove the variable, but then tell them how to set the variable. Should we remove the 2nd line?

Reply:
Yeah maybe bad wording. 🤔 ill give it a tweak.
Originally I just failed with no recourse, but decided to add the additional fallback. The phrasing probably got a bit muddled in that change.